src/structure/structure.ts

```diff
@@ -243,7 +248,7 @@ export class StructureViewer {
     private _structureForJSmol(index: number): string {
         if (this._cachedStructures[index] === undefined) {
             const s = this.loadStructure(index, this._structures[index]);
-            const check = checkStructure(s);
+            const check = checkStructure(s as unknown as JsObject);
```

Comment:
I don't understand this syntax.

Reply:
Here the compiler think that `s` is a `Structure`, but we want to check that. We can not cast `s as JsObject` since `Structure` and `JsObject` are too different, so we have to go through `unknown` first.